examples/json/json_shapes/README.md

```diff
@@ -67,27 +76,27 @@ Start two instances of RTI Shapes Demo:
   - Go to *Controls > Configuration*, and change the domain ID to `2`.
   - Subscribe to topic `Circle` with default options.
 
-In two separate command prompt windows for Routing Service instances that will
-run both scenarios described above, we run the following commands from the
-example directory (this is necessary to ensure the application find the
-configuration files):
+In two separate terminals, start two Routing Service instances:
 
-* ShapeTypeExtended to Content Container
+- `ShapeToJson`:
 
-```sh
-rtiroutingservice -cfgFile shapes_to_json_transf.xml -cfgName JsonTransformationExample
-```
+  ```sh
+  cd examples/json/json_shapes
+  rtiroutingservice -cfgFile RsJsonShapes.xml -cfgName ShapeToJson
```

Comment:
Actually, can't we consolidate this into one line?

```suggestion
  rtiroutingservice -cfgFile examples/json/json_shapes/RsJsonShapes.xml -cfgName ShapeToJson
```

Reply:
I would keep it in different lines so it is consistent with the rest of the examples. Some of the other examples use `USER_QOS_PROFILES.xml`, and therefore we need to run them from their specific folders.

Reply:
That's fine. I was just thinking it would be good to clue users to the fact that they can start RS from anywhere if they want to